quick-start/README.md

```diff
@@ -0,0 +1,169 @@
+# Quick Start
+
+The purpose of this walkthrough is to allow a quick test for this plugin.
+Some docker containers will be used to allow this test:
```

Comment:
We thought this was a DEV quick start, not generic quick start. A developer won't "test", they will setup their local environment (as you stated in the main README) and go from there. Suggestion:

```suggestion
# Dev Quick Start

The purpose of this guide is to help setting up a development environment for this plugin.
Some docker containers will be used:
```